app/api/business_routes.py

```diff
@@ -0,0 +1,101 @@
+from flask import Blueprint, jsonify, session, request
+from app.models import Business, db
+from flask_login import current_user, login_user, logout_user, login_required
+from app.forms.business_form import BusinessForm
+from app.api.auth_routes import validation_errors_to_error_messages
+
+business_routes = Blueprint('business', __name__)
+
+@business_routes.route("/<int:id>")
+def get_business_by_id(businessId):
+    """
+    Query for a business by id and return it as a dictionary
+    """
+    business = Business.query.get(businessId)
+
```

Comment:
Error response with status 404 is given when a business does not exist with the provided id